src/video_core/shader/shader_jit_x64.cpp

```diff
@@ -718,15 +718,19 @@ void JitShader::Compile_LOOP(Instruction instr) {
 
     looping = true;
 
+    // This block of code decodes the fields from the integer uniform at
+    // index instr.flow_control.int_uniform_id
+    // The Y (LOOPCOUNT_REG) and Z (LOOPINC) component are kept multiplied by 16 (Left shifted
+    // by 4 bit) to be used as an offset into the 16-byte vector registers later
```

Comment:
It's supposed to be a full stop [in the form of a new line]. I've had `instr.flow_control.int_uniform_id.` before but it just looks unintended due to the many dots in the field name itself.